mpa/cls/stage.py

```diff
@@ -89,6 +90,10 @@ def configure(self, model_cfg, model_ckpt, data_cfg, training=True, **kwargs):
     def configure_model(cfg, training, **kwargs):
         # verify and update model configurations
         # check whether in/out of the model layers require updating
+
+        if cfg.get('load_from', None) and cfg.model.backbone.get('pretrained', None):
```

Comment:
@harimkang How do you think about this order change? I changed to get cfg.load_from variable in configure_model. I couldn't find any error for both multi-class/multi-label tasks. Is there any problem that I missed you think?